src/components/Auth/Auth.tsx

```diff
@@ -38,6 +48,35 @@ function Auth() {
     setSignup(!signup);
   }, [signup]);
 
+  const handleRequestPasswordReset = useCallback(async () => {
+    try {
+      setResetting(true);
+
+      await resetPwForm.validateFields();
+
+      await UserAPI.requestResetPassword({ login: pwResetLogin });
+
+      showPwResetNotification();
+    } finally {
+      setResetting(false);
+    }
+  }, [resetPwForm, pwResetLogin]);
+
+  const handlePasswordReset = useCallback(() => {
+    setShowPwResetModal(true);
+  }, []);
+  const handleCancel = () => {
+    setShowPwResetModal(false);
+  };
+  const showPwResetNotification = () => {
+    notification.success({
+      message: 'Password Reset Email Has Been Sent',
+      description: 'We sent you an email with a reset link',
+      duration: 3.5,
+    });
+    setShowPwResetModal(false);
```

Comment:
This is not the right place to put this. Does the name `showPwResetNotification` imply in any way that it will close the modal? No. Just move this to `handleRequestPasswordReset`